internal/rukpak/convert/registryv1_test.go

```diff
@@ -167,22 +169,38 @@ var _ = Describe("RegistryV1 Suite", func() {
 		Context("Should generate objects successfully based on target namespaces", func() {
 			var (
 				svc             corev1.Service
-				csv             v1alpha1.ClusterServiceVersion
+				baseCSV         v1alpha1.ClusterServiceVersion
 				watchNamespaces []string
 			)
 
 			BeforeEach(func() {
-				csv = v1alpha1.ClusterServiceVersion{
+				baseCSV = v1alpha1.ClusterServiceVersion{
 					ObjectMeta: metav1.ObjectMeta{
 						Name: "testCSV",
 						Annotations: map[string]string{
 							"olm.properties": fmt.Sprintf("[{\"type\": %s, \"value\": \"%s\"}]", property.TypeConstraint, "value"),
 						},
 					},
 					Spec: v1alpha1.ClusterServiceVersionSpec{
-						InstallModes: []v1alpha1.InstallMode{{Type: v1alpha1.InstallModeTypeMultiNamespace, Supported: true}},
```

Comment:
I understand that it was removed because, after the conversion, we expect InstallStrategy, correct?

Reply:
This is the piece of the CSV that I want to configure in each of the test scenarios, so I removed it here and then each test scenario does a deep copy before setting InstallModes for that scenario.